controller/model/posture_check_model_mac.go

```diff
@@ -26,6 +26,25 @@ type PostureCheckMacAddresses struct {
 	MacAddresses []string
 }
 
+func (p *PostureCheckMacAddresses) Evaluate(pd *PostureData) bool {
+	if pd.Mac.TimedOut {
+		return false
+	}
+
+	validAddresses := map[string]struct{}{}
```

Comment:
same comment wrt creating the validAddresses map - can it be done outside the Evaluate?

Reply:
The PostureCheckMacAddresses `p` is coming from bbolt and I don't see a great place to put this logic such that it will improve anything. 

Something I wanted to add, but didn't in this interaction, was an evaluation cache that would be smart enough to know if the data actually updated or if it just refreshed. That would actually be a bigger improvement.